platform/mellanox/mlnx-platform-api/sonic_platform/thermal.py

```diff
@@ -119,6 +119,17 @@
             "name": "Ambient Switch Board Temp",
             "temperature": "swb_amb",
             "default_present": False
+        },
```

Comment:
Maybe you need define thermal capability in device_data.py for this sensor?

Reply:
so to add it to all the platforms? 
or maybe to add it as "default_present": False ? 
you mean to "PCH Temp" right?

Reply:
we probably need add it to platform that support this sensor

Reply:
Moose will support it , currently Moose is not added as a platform right?